API/BookSharing/BookSharing.Infrastructure/BookApi/IBookService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Threading.Tasks;
+
+namespace BookSharing.Infrastructure.Interface
+{
+    public interface IBookService
```

Comment:
inna nazwa, to nie jest normalny, typowy serwis, to interfejs któy dostarcza nam informacje o ksiazce z zewnetrznego zrodła